delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/wiring/RealmsModule.scala

```diff
@@ -44,7 +45,7 @@ object RealmsModule extends ModuleDef {
         hc: HttpClient
     ) =>
       val wellKnownResolver = WellKnownResolver((uri: Uri) => hc[Json](HttpRequest(uri = uri))) _
-      RealmsImpl(cfg, wellKnownResolver, eventLog)(as, Clock[UIO])
+      RealmsImpl(cfg, wellKnownResolver, eventLog)(as, Scheduler.global, Clock[UIO])
```

Comment:
the scheduler is bound in DeltaModule, add it to the param list of the fn instead of making it constant.